src/Symfony/Component/Notifier/Bridge/JoliNotif/JoliNotifTransport.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\JoliNotif;
+
+use Joli\JoliNotif\DefaultNotifier as JoliNotifier;
+use Joli\JoliNotif\Notification as JoliNotification;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Exception\RuntimeException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\DesktopMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * @author Ahmed Ghanem <ahmedghanem7361@gmail.com>
+ */
+final class JoliNotifTransport extends AbstractTransport
+{
+    public function __construct(
+        private readonly JoliNotifier $joliNotifier,
+        ?EventDispatcherInterface $dispatcher = null,
+    ) {
+        parent::__construct(dispatcher: $dispatcher);
```

Comment:
```suggestion
        parent::__construct($dispatcher);
```

Reply:
$dispatcher is defined as 2nd parameter in the parent constructor, So I may keep it as it or do `parent::__construct(null, $dispatcher);`

Reply:
yes, we'd better not use named arguments since they're not part of the BC promise (except for attributes)